### PR TITLE
add new paper

### PR DESCRIPTION
**Pull request by ElioChan**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.

Add a new paper：Structure-Aware Private Set Intersection with Applications to Fuzzy matching.
It introduce new generic paradigm for structrue-aware Psi based on function secret-sharing(FSS) which improve the performance when one of the party's data strucutre is public known.  And  it develop several improved FSS techniques and explore Sa-PSI in depth a natural application of it, such as Fuzzy matching.

### CODE REVIEW

**Comment by ElioChan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 1]
```diff
@@ -120,3 +122,12 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
 - PIR-PSI: Scaling Private Contact Discovery
   *Daniel Demmler, Peter Rindal, Mike Rosulek, Ni Trieu*
   PETS 2018, [eprint](https://eprint.iacr.org/2018/579), DRRT18
+
+
+- Structure-Aware Private Set Intersection with Applications to Fuzzy Matching
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:127]: 尾部可以加两个空格吗？会渲染的更好看一些

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 2]
```diff
@@ -120,3 +122,12 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
...
+  *Gayathri Garimella, Mike Rosulek, Jaspal Singh*
+  CRYPTO 2022, [eprint](https://eprint.iacr.org/2022/1011.pdf), GRS22
+
+- Malicious Secure, Structure-Aware Private Set Intersection
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:131]: ditto

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 3]
```diff
@@ -120,3 +122,12 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
 - PIR-PSI: Scaling Private Contact Discovery
   *Daniel Demmler, Peter Rindal, Mike Rosulek, Ni Trieu*
   PETS 2018, [eprint](https://eprint.iacr.org/2018/579), DRRT18
+
+
+- Structure-Aware Private Set Intersection with Applications to Fuzzy Matching
+  *Gayathri Garimella, Mike Rosulek, Jaspal Singh*
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:128]: ditto

**Review by Jamie-Cui — CHANGES_REQUESTED** [review]

File: docs/awesome-pets/papers/applications/set/psi.md (hunk 1 above)
Comment (ElioChan) [reply docs/awesome-pets/papers/applications/set/psi.md]: 好的

**Review by Jamie-Cui — APPROVED** [review]
LGTM

**Pull request merged**